indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,134 @@
         }
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+
+    .contribution-row {
+        @include border-left($width: 3px);
+        padding-left: 10px;
+        margin-bottom: 1em;
+
+        &:hover {
+            border-color: $blue;
+        }
+        .row-title {
+            @include border-bottom();
+            font-size: 14pt;
+        }
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+        .label {
+            border: 0;
+            margin-top: 7px;
+            display: inline-block;
+            padding: 5px 7px;
+
+            &.review-role {
+                @include border-all($gray);
+                margin-right: 10px;
+                background-color: white;
+                color: $light-black;
+                i {
+                    color: $blue;
+                }
+            }
+
+            &.reviews {
+                margin-top: 0;
+                border-width: 1px;
+                border-style: solid;
+                //border: 1px solid;   not used not useed
+                border-bottom-right-radius: 0;
+                border-top-right-radius: 0;
+            }
+            .icon {
+                //    not used
+                display: inline-block;
+                margin-right: 7px;
+            }
+
+            &.submitted {
+                background-color: $blue;
+            }
+
+            &.accepted {
+                background-color: $green;
+            }
+
+            &.rejected {
+                background-color: $red;
+            }
+
+            &.to_be_corrected {
```

Comment:
You can set the corresponding css class names separately on enums once this PR is merged:

https://github.com/indico/indico/pull/2793/commits/8df8650b44216d9a0f55d0776053cfb449b9f8dd